programs/program-statistics.md

```diff
@@ -1,10 +1,14 @@
 # Program Statistics
 
+These are the number of successful internships per year for each program.
```

Comment:
Are these actually mentees that passed? Just assumed they are.

Would we want to also list number of passed mentees / failed mentees broken down?
We need to check if Google / LFX publishes break down of passes/failures. If they don't, maybe we shouldn't do the break down and simply have the numbers we have right now.